modules/ml/src/main/java/org/apache/ignite/ml/composition/ModelsComposition.java

```diff
@@ -75,4 +76,18 @@ public PredictionsAggregator getPredictionsAggregator() {
     public List<Model<Vector, Double>> getModels() {
         return models;
     }
+
+    /** {@inheritDoc} */
+    @Override public String toString() {
+        return toString(false);
+    }
+
+    /** {@inheritDoc} */
+    @Override public String toString(boolean pretty) {
+        return new StringBuilder("ModelsComposition [").append(pretty ? "\n\t" : "")
+            .append("aggregator = ").append(predictionsAggregator.toString(pretty)).append(pretty ? "\n\t" : ", ")
+            .append("models = [").append(pretty ? "\n\t\t" : "")
+            .append(models.stream().map(x -> x.toString(false)).collect(Collectors.joining(pretty ? "\n\t\t" : ", ")))
+            .append(pretty ? "\n\t]\n]" : "]]").toString();
```

Comment:
Could we say that there are a lot of magic char sequences? Could you move it to constatns or util file?